File: 100-Days-Series/Day 70/ArrayTarget.js

// Arrow Funtion to find two numbers that add up to a specific target
const getFind=(arr,target)=>{
    for(let i=0;i<arr.length;i++){
        for(let j=i+1;j<arr.length;j++){
            if(arr[i]+arr[j]==target){
                console.log("Indices : "+i+", "+j);
                return;
            }
        }
    }
    console.log("No Solution Found");
}
// Let's Start Here
const arr=new Array(2,3,4,5,6,10,22);
let target=11;
getFind(arr,target);

File: 100-Days-Series/Day 70/ArrayTarget.py

# Funtion to find two numbers that add up to a specific target
def get_find(arr,target):
    for i in range (0,len(arr)):
        for j in range(i+1,len(arr),1):
            if arr[i]+arr[j]==target:
                print('Indices :',i,', ',j)
                return 
            
    print('No Solution Found')

# Let's Start Here
arr=[2,3,5,7,8,12,19]
target=15
get_find(arr,target)